.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,12 +16,3 @@ Fixes #
 **Special notes for your reviewer**:
 
 _Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._
```

Comment:
Sorry didn't see your PR. Can you drop the same as I did in https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/2108?